test/operate/test_pdb.py

```diff
@@ -0,0 +1,38 @@
+import numpy
+import logging
+import pkg_resources
+import os
+
+from pdb2sql import pdb2sql
+from nose.tools import ok_
+
+from deeprank.operate.pdb import get_residue_contact_atom_pairs
+
+
+_log = logging.getLogger(__name__)
+
+def test_residue_contact_atoms():
+
+    pdb_path = os.path.join(pkg_resources.resource_filename(__name__, ''),
+                            "../1AK4/native/1AK4.pdb")
+
+    try:
+        pdb = pdb2sql(pdb_path)
+
+        contact_atom_pairs = get_residue_contact_atom_pairs(pdb, 'C', 145, 8.5)
+
+        contact_atoms = set([])
+        for atom1, atom2 in contact_atom_pairs:
+            contact_atoms.add(atom1)
+            contact_atoms.add(atom2)
+
+        contact_atom_names = [tuple(x) for x in pdb.get("chainID,resSeq,name", rowID=list(contact_atoms))]
+    finally:
+        pdb._close()
+
+    neighbour = ('C', 144, 'CA')
+    distant = ('B', 134, 'OE2')
+
```

Comment:
I am sorry, I did not understand the above two lines on "neighbour" and "distant"

Reply:
Let me explain: I wanted to test the contact detecting function, so I selected some PDB residues. In yasara, I selected a residue to find the contacts for. It's chain C residue number 545. (subtract 400 for pdb2sql, who numbers differently) A residue quite close to this is the "neighbour" chain C residue 544. (again, subtract 400)
But the residue in chain B number 134 is far away from the other selected residues and should not be found. This test checks whether that's the case.
I will add some explaining comments ..

Reply:
Thanks Coos. I suspected that is what you are trying to do here. 
I did not know about the numbering discrepancy